front/production_checks/lib/libcheck.ts

```diff
@@ -0,0 +1,10 @@
+export type CheckFunction = (
```

Comment:
no need to repeat lib?

Reply:
I renamed it from `/lib/libcheck.ts` to `types/check.ts`